gui/src/map/Tile.java

```diff
@@ -23,6 +25,37 @@ public class Tile {
     // ----- M E T H O D S -----
 
 
-    // ----- S E T T E R S -----
+    // ----- G E T T E R S - S E T T E R S -----
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
```

Comment:
Du coup getter et setter ils sautent aussi

Reply:
should be good now